src/pages/Cart/Cart.js

```diff
@@ -23,6 +24,11 @@ const Cart = () => {
     useSelector((state) => state.userReducer.user?.budget) || 0;
   const dispatch = useDispatch();
   const history = useHistory();
+
+  useLayoutEffect(() => {
+    scrollTo(220);
```

Comment:
use not static value, it's can be changed on different screen dimensions. It will be better to use offset values of an element. Maybe the bottom position of a page title element